spec/messenger_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'messenger'
+
+describe Messenger do
+  subject(:messenger) { described_class.new }
+  let(:client) { double :client }
+  let(:phone) { Dotenv.load['TWILIO_DESTINATION_PHONE'] }
+
+  describe '#send_text' do
+    it 'sends a text with a message' do
+      message = 'Hello!'
+      twilio_message_body = { from: Dotenv.load['TWILIO_PHONE'], to: Dotenv.load['TWILIO_DESTINATION_PHONE'], body: message }
```

Comment:
Metrics/LineLength: Line is too long. [125/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)